storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/RocksDbInstanceFactory.java

```diff
@@ -164,24 +165,71 @@ private static DBOptions createDBOptions(
   }
 
   private static ColumnFamilyOptions createColumnFamilyOptions(
-      final KvStoreConfiguration configuration, final Cache cache) {
-    return new ColumnFamilyOptions()
-        .setCompressionType(configuration.getCompressionType())
-        .setBottommostCompressionType(configuration.getBottomMostCompressionType())
-        .setLevelCompactionDynamicLevelBytes(true)
-        .setTableFormatConfig(createBlockBasedTableConfig(cache));
+      final KvStoreConfiguration configuration, final KvStoreColumn<?, ?> column) {
+    final ColumnFamilyOptions cfOptions;
+    try {
+      final LRUCache cache =
+          column
+              .getIsLargerCacheAvalilable()
+              .map(
+                  isLarger -> {
+                    if (isLarger) {
+                      LOG.info("Using larger cache for column {}", column.getId().toHexString());
```

Comment:
NIT: not sure if we need this as a info. It seems to be pretty low level and I belive we are happy that most users are gonna use default values anyway.

Reply:
This was just for testing purposes.